obuilder.opam

```diff
@@ -22,7 +22,7 @@ depends: [
   "ppx_sexp_conv"
   "sha"
   "sqlite3"
-  "dockerfile"
+  "dockerfile" {>= "6.6.0"}
```

Comment:
This should be mainly for `obuilder-spec`

Reply:
Oops - fixed!